### ISSUE TITLE
Unable to start Rider

### ISSUE THREAD

**Issue by safesparrow** [JetBrains/resharper-fsharp#439]
Hi,
I'm trying to run Rider with the `net223` branch, and I'm getting lots of Java runtime errors.
If it matters, I have these JDK versions installed:
- 19
- 18.0.21
- 16
and `JAVA_HOME` pointing to one of them (doesn't matter which) - as I understand though the plugin is using its own JDK.

`./gradleW runIDE` is throwing the following:
```
2022-11-10 00:39:22,003 [   1327]   WARN - o.j.n.NativeTrustedRootsInternalUtils - Unable to get custom trusted certificates list from the operating system: Could not initialize class com.sun.jna.Native
java.lang.NoClassDefFoundError: Could not initialize class com.sun.jna.Native
        at org.jetbrains.nativecerts.win32.Crypt32Ext.<clinit>(Crypt32Ext.java:64)
...
2022-11-10 00:39:23,403 [   2727] SEVERE - #c.j.r.p.RiderWindowsRegistryHandler - Unable to run update Rider registry key
java.lang.NoClassDefFoundError: Could not initialize class com.sun.jna.Native
        at com.sun.jna.platform.win32.WinReg$HKEYByReference.<init>(WinReg.java:52)
        at com.sun.jna.platform.win32.WinReg$HKEYByReference.<init>(WinReg.java:48)
...

2022-11-10 00:39:23,637 [   2961] SEVERE - #c.i.i.ApplicationLoader - cannot execute preloading activity com.intellij.ide.plugins.DynamicPluginVfsListenerInitializer
java.lang.IllegalAccessError: class com.intellij.openapi.vfs.impl.local.LocalFileSystemImpl (in unnamed module @0x16a4033a) cannot access class sun.nio.fs.BasicFileAttributesHolder (in module java.base) because module java.base does not export sun.nio.fs to unnamed module @0x16a4033a
        at com.intellij.openapi.vfs.impl.local.LocalFileSystemImpl.listWithAttributes(LocalFileSystemImpl.java:285)
        at com.intellij.openapi.vfs.impl.local.LocalFileSystemImpl.lambda$new$0(LocalFileSystemImpl.java:42)

...
2022-11-10 00:39:23,640 [   2964] SEVERE - #c.i.i.ApplicationLoader - JetBrains Rider 2022.3 EAP 9 D  Build #RD-223.7563.27
2022-11-10 00:39:23,640 [   2964] SEVERE - #c.i.i.ApplicationLoader - JDK: 17.0.5; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2022-11-10 00:39:23,640 [   2964] SEVERE - #c.i.i.ApplicationLoader - OS: Windows 11
2022-11-10 00:39:24,788 [   4112] SEVERE - #c.i.o.a.i.FlushQueue - Could not initialize class com.sun.jna.Native
java.lang.NoClassDefFoundError: Could not initialize class com.sun.jna.Native
        at com.sun.jna.platform.win32.WinReg$HKEYByReference.<init>(WinReg.java:52)
        at com.sun.jna.platform.win32.WinReg$HKEYByReference.<init>(WinReg.java:48)
```

I can see this `javac.exe` file fetched by gradle during `./gradlew prepare` or `./gradlew runIde`:
```
%USER_PROFILE%\.gradle\caches\modules-2\files-2.1\com.jetbrains\jbre\jbr_jcef-17.0.5-windows-x64-b653.6\extracted\jbr_jcef-17.0.5-windows-x64-b653.6\bin\javac.exe
```

What am I doing wrong?

**Comment by En3Tho** [comment]
Same here.. I'm not sure which certificate I have to install or what is it

**Comment by auduchinok** [comment]
It has been just fixed in `gradle-intellij` 1.10, please update to a newer `net231` that uses this version.
